DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -453,7 +454,7 @@ namespace reco {
     HitPattern hitPattern_;
 
     /// perigee 5x5 covariance matrix
-    float covariance_[covarianceSize];
+    Float16_t covariance_[covarianceSize];  //[0,0,10]
```

Comment:
to reduce confusion, it may be worth to be explicit and add a comment that `Float16_t` is a full float in memory and is streamed as char (8 bit) exponent and 12 active bits for mantissa 
https://root.cern.ch/doc/master/TBufferFile_8cxx_source.html#l00590

The double32 is a bit less confusing